src/query/procedure/py_module.cpp

```diff
@@ -1062,17 +1063,35 @@ void CallPythonCleanup(const py::Object &py_cleanup) {
 void CallPythonInitializer(const py::Object &py_initializer, mgp_list *args, mgp_graph *graph, mgp_memory *memory) {
   auto gil = py::EnsureGIL();
 
+  auto error_to_msg = [](const std::optional<py::ExceptionInfo> &exc_info) -> std::optional<std::string> {
+    if (!exc_info) return std::nullopt;
+    // Here we tell the traceback formatter to skip the first line of the
+    // traceback because that line will always be our wrapper function in our
+    // internal `mgp.py` file. With that line skipped, the user will always
+    // get only the relevant traceback that happened in his Python code.
+    return py::FormatException(*exc_info, /* skip_first_line = */ true);
+  };
+
   auto call = [&](py::Object py_graph) -> std::optional<py::ExceptionInfo> {
     py::Object py_args(MgpListToPyTuple(args, py_graph.Ptr()));
     if (!py_args) return py::FetchError();
     auto py_res = py_initializer.Call(py_graph, py_args);
-    return {};
+    if (!py_res) return py::FetchError();
+    return std::nullopt;
   };
 
+  std::optional<std::string> maybe_msg;
```

Comment:
Isn't a better name here `maybe_err`?

Reply:
Ah, I was consistent just with the rest of the file.